pkg/api/api.go

```diff
@@ -10,9 +10,10 @@ type Metadata struct {
 	Annotations map[string]string `json:"annotations"`
 	Labels      map[string]string `json:"labels"`
 
-	CreatedAt  time.Time  `json:"createdAt"`
-	DeletedAt  *time.Time `json:"deletedAt,omitempty"`
-	Generation int64      `json:"generation"`
+	CreatedAt       time.Time  `json:"createdAt"`
+	DeletedAt       *time.Time `json:"deletedAt,omitempty"`
+	FirstShutdownAt time.Time  `json:"firstShutdownAt"`
```

Comment:
```suggestion
	ShutdownAt time.Time  `json:"ShutdownAt"`
```

Reply:
Presently, we are initiating machine shutdowns at regular intervals. However, in this field, we are preserving the timestamp when the initial shutdown attempt was made. Consequently, I considered that `FirstShutdownAt` might be a more fitting name for this field. Please share your thoughts on this.